index.html

```diff
@@ -2,8 +2,8 @@
 <html>
     <head>
         <meta charset="UTF-8">
-        <title>Numbers</title>        
         <link rel="stylesheet" href="css/styles.css">        
+        <title>Numbers</title>        <!-- Comment -->
```

Comment:
There are a lot of places in your project where there are extra spaces at the end of lines.  Most editors can clean that up automatically for you.  If you're using VSCode, you can have it do so automatically on saving by setting the `files.trimTrailingWhitespace` configuration option.